modules/deploy/pages/deployment-option/cloud/create-dedicated-cloud-cluster-aws.adoc

```diff
@@ -3,14 +3,11 @@
 :page-aliases: cloud:create-dedicated-cloud-cluster-aws.adoc, deploy:deployment-option/cloud/provision-a-dedicated-cluster/index.adoc
 :page-cloud: true
 
-To create a Dedicated Cloud cluster, log in to https://cloud.redpanda.com[Redpanda Cloud^]. Enter your organization name, and click *Continue*. 
+After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the *Clusters* page, which lists all the clusters in your organization. When you create new clusters, if your organization has more than one namespace, you select the namespace where you want to create the new cluster. 
 
 == Create a Dedicated cluster
 
-. On the Home page, select your namespace, or create a new one. 
-+
-Namespaces help you organize clusters and resources. For example, you can create namespaces for different teams or projects. A default namespace is created with every organization. When an organization has more than one namespace, you can select the namespace where you want to create a new cluster. 
-. Click *Create cluster*, then click *Create Dedicated cluster*.
+. On the Clusters page, click *Create cluster*, then click *Create Dedicated cluster*.
 +
 Enter a cluster name, and select the provider (AWS or GCP), region, availability zone type (single AZ or multi AZ) and zones, throughput tier, and Redpanda version.
```

Comment:
@micheleRP We should also add the bit about selecting a namespace for Dedicated, as we do for BYOC and Serverless.

Reply:
fixed, thanks @frankieshakes!